migrations.go

```diff
@@ -53,14 +56,39 @@ func migrate_4_SwarmKey(cfg *Config) bool {
 // for bootstrap nodes.
 // Replaces all bootstrap nodes with default values if so.
 func migrate_5_Bootstrap_node(cfg *Config) bool {
-	obsoleteBootstrapNodeList := []string{"3.120.224.94"}
+	obns := []string{
+		"3.120.224.94",
+	}
+	peers, _ := DefaultBootstrapPeers()
+	return doMigrateNodes(cfg, obns, peers)
+}
+
+func migrate_6_EnableAutoRelay(cfg *Config) bool {
+	if cfg.Swarm.EnableAutoRelay != DefaultEnableAutoRelay {
+		cfg.Swarm.EnableAutoRelay = DefaultEnableAutoRelay
```

Comment:
This will *always* turn it off.. migration is run on each start of daemon...

Reply:
So it has to be inside either bootstrap node migrations.

Reply:
i guess there is really no harm turning it on by force unless some devs are mad about it